src/boot.rs

```diff
@@ -70,6 +70,7 @@ pub async fn start(boot: BootResult) -> Result<()> {
             serve(router, &app_context.config).await?;
         }
         (Some(router), None) => {
+            println!("Config: {:?}", app_context.config);
```

Comment:
I believe it is not relevant here. 
In additionally, be cautious about printing/logging all the configuration details, as it may include sensitive data that you wouldn't want to expose in logs or prints.

Reply:
This was an artifact from debugging, it has been removed, thank you